### ISSUE TITLE
Streaming does not send all new records

### ISSUE THREAD

**Issue by umitanuki** [alpacahq/marketstore#64]
While I'm testing the client code, I noticed that if more than one record are written per symbol, only one is sent through the stream.

**Comment by rocketbitz** [comment]
Oh you mean for batch writes?

**Comment by umitanuki** [comment]
Yes.  It is controversial but I was expecting so.  Even in the case of
incremental inserts, there might be a two bars written in one op

Hitoshi Harada


On May 29, 2018 at 08:26, Christopher Ryan <notifications@github.com> wrote:


Oh you mean for batch writes?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/alpacahq/marketstore/issues/64#issuecomment-392819499>,
or mute the thread
